ItemRack/ItemRack.lua

```diff
@@ -78,7 +100,7 @@ ItemRackItems = {
 }
 
 ItemRack.NoTitansGrip = {
-	["Polearms"] = 1,
+	--["Polearms"] = 1, -- 3.3.5 / Wrath Classic support Polearm Titan's Grip
```

Comment:
They fixed Titan's Grip in 3.4.1 so you can no longer dual-wield polearms.

Reply:
Good to know, I'll revert the change then in next push.